libraries/AP_Compass/CompassCalibrator.cpp

```diff
@@ -1065,14 +1065,15 @@ bool CompassCalibrator::calculate_orientation(void)
  */
 bool CompassCalibrator::fix_radius(void)
 {
-    if (AP::gps().status() < AP_GPS::GPS_OK_FIX_2D) {
+    Location loc;
+    if (!AP::ahrs().get_location(loc)) {
+        gcs().send_text(MAV_SEVERITY_WARNING, "MagCal: No location, fix_radius skipped");
```

Comment:
We should probably use GCS_SEND_TEXT and I think we should check the prefix is consistent with other text messages from this library.